CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # CHANGELOG
 ## main (unreleased)
 - Work in Progress
+- If a route path does not start with "^" match the entire path
```

Comment:
Might be good to prefix with `Fix: `

Probably a wider discussion on what we want to use to have a consistent/readable `CHANGELOG`.

`fix, feature, improvement` 🤷‍♀️

Reply:
I guess I will wait on _CONTRIBUTIONS_ file with guides on keywords in commits and changelog :)

Reply:
Good call!